### DISCUSSION TITLE
能不能建个qq群或者电报群之类的

### DISCUSSION THREAD

**Discussion by blibilijojo** [Richasy/Bili.Uwp#1266] [Polls]
emm，用了这么久，不仅是云之欢大大感觉孤独，其实用的也很孤独的，感觉这样没大有活力
所以。。。。
建个群吧

**Comment by Richasy** [comment]
孤独倒还好，我更怕的是没有效率。
能上Github的多多少少知识水平是达标的，反馈问题或者讨论哪怕少点，至少也能对项目有所助益。建群的话只怕会陷入到无尽的解释之中，太过分散精力。
我鼓励爱好者建群，也可以拉我（专业潜水摸鱼），但我自己应该是不会主导再去建群了

**Comment by blibilijojo** [comment]
哦

**Comment by YanSu5001** [comment]
有爱好者建的话就踢我一下。
